### PR TITLE
Document ReduxInjector

### LINKED ISSUE #9: Add ReduxInjector to the readme
Missing ReduxInjector docs in the readme.

### PR DESCRIPTION
**Pull request by maier49**
The following has been addressed in the PR:

* [x] There is a related issue
* [x] All code matches the [style guide](https://github.com/dojo/meta/blob/master/STYLE.md)
* [ ] Unit or Functional tests are included in the PR

**Description:**
Adds the redux injector documentation from the PR to the README

Resolves #9

### CODE REVIEW

**Review by dylans — APPROVED** [review]

**Comment by agubler** [comment]
@maier49 the example on the PR is now slightly out of date, I'll comment on the diffs in the morning :)

**Comment by agubler** [comment]
This is a real world usage that is up to date https://github.com/dojo/examples/blob/master/todo-mvc-tsx/src/main.ts

**Comment by maier49** [comment]
@agubler I updated the example, and also the wording, based on that example. Let me know if the updated wording needs any tweaks.

**Review by kitsonk — APPROVED** [review]
I don't know enough about the ReduxInjector, so it looks good from my point of view but defer to @agubler

File: README.md [hunk 1]
```diff
@@ -55,7 +55,50 @@ For most existing Dojo 1 Dijits, the TypeScript typings can be found at [dojo/ty
 
 ### ReduxInjector
 
-*Coming Soon*
+`ReduxInjector` can be used to bind a redux store and Dojo 2 widgets using the `registry`.
+
+An injector can be defined in the registry, which is then registered with the `Router`, and provided to the `Projector` as one of its properties. This is demonstrated in the example below.
```
Comment (agubler) [inline README.md]: I don't understand this sentence, the `Router` doesn't have anything to do with the using the `ReduxInjector`...

File: README.md [hunk 2]
```diff
@@ -55,7 +55,50 @@ For most existing Dojo 1 Dijits, the TypeScript typings can be found at [dojo/ty
...
+	currentTodo: '',
+	activeCount: 0,
+	completedCount: 0
+};
+
+const registry = new Registry();
+const store = createStore(todoReducer, defaultState, global.__REDUX_DEVTOOLS_EXTENSION__ && global.__REDUX_DEVTOOLS_EXTENSION__());
+registry.defineInjector('application-state', new ReduxInjector(store));
+
+const config = [
```
Comment (agubler) [inline README.md]: Can drop anything router related from the examples (might be confusing)

File: README.md [hunk 3]
```diff
@@ -55,7 +55,50 @@ For most existing Dojo 1 Dijits, the TypeScript typings can be found at [dojo/ty
...
+
+const defaultState = {
+	todos: [],
+	currentTodo: '',
+	activeCount: 0,
+	completedCount: 0
+};
+
+const registry = new Registry();
+const store = createStore(todoReducer, defaultState, global.__REDUX_DEVTOOLS_EXTENSION__ && global.__REDUX_DEVTOOLS_EXTENSION__());
```
Comment (agubler) [inline README.md]: might want to call out that this initialises the redux dev tools extension (and maybe a link to it?).

**Comment by maier49** [comment]
@agubler Updated again

File: README.md (hunk 3 above)
Comment (matt-gadd) [reply README.md]: do we need this in the readme?

Comment (agubler) [reply README.md]: Nope probably not to be honest.

Comment (maier49) [reply README.md]: I removed the redux dev tools extension from the example code and removed the explanation about it.

**Review by agubler — APPROVED** [review]

**Pull request merged**